scripts/handle-release.sh

```
@@ -184,6 +181,8 @@ else
 fi
 yarn workspaces foreach --all --topological --no-private run build
 
+echo "Publishing in order: ${PUBLISH_ORDER[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)